.github/workflows/build_packages.yaml

```diff
@@ -35,6 +35,7 @@ jobs:
       - name: Get profile to build
         id: get_profile
         run: |
+          git config --global --add safe.directory "$GITHUB_WORKSPACE"
```

Comment:
Maybe:

```suggestion
          git config --global --add safe.directory "*"
```
?

Reply:
I try to follow the "Principle of least privilege" as much as possible.

Reply:
If this one does not work, I'd like to know why it's not enough.